includes/Core/Assets/Assets.php

```diff
@@ -124,21 +124,30 @@ function() {
 			}
 		);
 
-		add_action(
-			'enqueue_block_editor_assets',
-			function() {
-				$assets = $this->get_assets();
+		// A map of asset contexts to load for specific hooks.
+		$actions_contexts_map = array(
+			'admin_print_scripts-edit.php' => Asset::CONTEXT_ADMIN_POSTS,
```

Comment:
We need to make sure that we load assets with the `CONTEXT_ADMIN_POSTS` context only on the posts list table (`$GLOBALS['typenow'] === 'post'`) and not for post list tables of other custom post types.

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/08c3d3a8da6c25bac7a7a0b4ec579fd05e30d7a7